pkg/controller/core/localqueue_controller.go

```diff
@@ -84,6 +84,13 @@ func (r *LocalQueueReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	}
 
 	queueObj.Status.PendingWorkloads = pending
+	admitted, errAdmitted := r.cache.AdmittedWorkloadsLocalQueue(&queueObj)
```

Comment:
Looks good, but this needs to be tested.

You can add checks to the existing integration tests in `test/integration`

Reply:
Thank you!  I was having some trouble running those on my setup.  

I had some failures in unrelated code and some stuff to do with certs.  I'll look more into it and add some tests in test/integration also.

Reply:
Did you try `make test-integration`?

Reply:
Yea, I'll try it again.

Reply:
Alright.  There is a regression caused by this code so I'll look into it, fix it and add some tests here.    

I am unable to run `make test-integration` on my Mac M1.  it says architecture not supported and then fails every test.  

I get failures when I get a test failure but it seems to be caused by the controller test detecting an error.

Reply:
@kannon92 Hi.
We will fix `make test-integration` for M1 Mac in #362. If you want to run `test-integration` on your local now, please check [this comment](https://github.com/kubernetes-sigs/kueue/pull/362#discussion_r957756617).

Reply:
@tenzen-y Thank you for that suggestion!  Applied that fix locally and I was able to deduce where my test failure was.  Pushed up a fix for that.

Reply:
It looks like existing tests in the controller do verify status so I just added a new field for AdmittedWorkloads in the test.  I think this is fine.